content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,289 @@
+title: Node Autoscaling
+linkTitle: Node Autoscaling
+description: >-
+  Automatically provision and consolidate the Nodes in your cluster to adapt to demand and optimize cost.
+content_type: concept
+weight: 15
+---
+
+In order to run workloads in your cluster, you need
+{{< glossary_tooltip text="Nodes" term_id="node" >}}. Nodes in your cluster can be _autoscaled_ -
+dynamically [_provisioned_](#provisioning), or [_consolidated_](#consolidation) to provide needed
+capacity while optimizing cost. Autoscaling is performed by Node [_autoscalers_](#autoscalers).
+
+## Node provisioning {#provisioning}
+
+If there are Pods in a cluster that can't be scheduled on existing Nodes, new Nodes can be
+automatically added to the cluster&mdash;_provisioned_&mdash;to accommodate the Pods. This is
+especially useful if the number of Pods changes over time, for example as a result of
+[combining horizontal workload with Node autoscaling](#horizontal-workload-autoscaling).
+
+Autoscalers provision the Nodes by creating and deleting cloud provider resources backing them. Most
+commonly, the resources backing the Nodes are Virtual Machines.
+
+The main goal of provisioning is to make all Pods schedulable. This goal is not always attainable
+because of various limitations, including reaching configured provisioning limits, provisioning
+configuration not being compatible with a particular set of pods, or the lack of cloud provider
+capacity. While provisioning, Node autoscalers often try to achieve additional goals (for example
+minimizing the cost of the provisioned Nodes or balancing the number of Nodes between failure
+domains).
+
+There are two main inputs to a Node autoscaler when determining Nodes to
+provision&mdash;[Pod scheduling constraints](#provisioning-pod-constraints),
+and [Node constraints imposed by autoscaler configuration](#provisioning-node-constraints).
+
+Autoscaler configuration may also include other Node provisioning triggers (for example the number
+of Nodes falling below a configured minimum limit).
+
+{{< note >}}
+Provisioning is formerly known as _scale-up_ in Cluster Autoscaler.
+{{< /note >}}
+
+### Pod scheduling constraints {#provisioning-pod-constraints}
+
+Pods can express [scheduling constraints](/docs/concepts/scheduling-eviction/assign-pod-node/) to
+impose limitations on the kind of Nodes they can be scheduled on. Node autoscalers take these
+constraints into account to ensure that the pending Pods can be scheduled on the provisioned Nodes.
+
+The most common kind of scheduling constraints are the resource requests specified by Pod
+containers. Autoscalers will make sure that the provisioned Nodes have enough resources to satisfy
+the requests. However, they don't directly take into account the real resource usage of the Pods
+after they start running. In order to autoscale Nodes based on actual resource usage, you can
+combine [horizontal workload autoscaling](#horizontal-workload-autoscaling) with Node autoscaling.
```

Comment:
This sentence slightly worries me that people will read `based on actual resource usage` as the classic ASG example of scaling on CPU usage of the VMs. Maybe make clear that it's purely workload resource usage here?

Reply:
I don't see that this is done @towca